sound/soc/sof/intel/hda.h

```diff
@@ -934,6 +935,9 @@ struct hda_dai_widget_dma_ops {
 		       struct snd_pcm_substream *substream, int cmd);
 	int (*post_trigger)(struct snd_sof_dev *sdev, struct snd_soc_dai *cpu_dai,
 			    struct snd_pcm_substream *substream, int cmd);
+	void (*codec_dai_set_hext_stream)(struct snd_sof_dev *sdev,
```

Comment:
you probably mean: `codec_dai_set_hdac_stream` or `codec_dai_set_hstream` or just simply `codec_dai_set_stream`?

Reply:
yes, this can be simplified, and we can avoid passing the hext_stream when we only need the hdac_stream.

Thanks for the suggestion @ujfalusi

Reply:
done